src/utils/utils/utils.ts

```diff
@@ -3,24 +3,27 @@ import { IoK8sApiCoreV1Service } from '@kubevirt-ui/kubevirt-api/kubernetes';
 export const isEmpty = (obj) =>
   [Object, Array].includes((obj || {}).constructor) && !Object.entries(obj || {}).length;
 
-export const get = (obj, path, defaultValue = undefined) => {
-  const travel = (regexp) =>
+export const get = (obj: unknown, path: string | string[], defaultValue = undefined) => {
+  const travel = (regexp: RegExp) =>
     String.prototype.split
       .call(path, regexp)
       .filter(Boolean)
-      .reduce((res, key) => (res !== null && res !== undefined ? res[key] : res), obj);
+      .reduce((res: { [x: string]: any }, key: string | number) => {
+        return res !== null && res !== undefined ? res[key] : res;
+      }, obj);
   const result = travel(/[,[\]]+?/) || travel(/[,[\].]+?/);
   return result === undefined || result === obj ? defaultValue : result;
 };
 
 export const isUpstream = (window as any).SERVER_FLAGS?.branding === 'okd';
 
-export const isString = (val) => val !== null && typeof val === 'string';
+export const isString = (val: unknown) => val !== null && typeof val === 'string';
 
 export const getSSHNodePort = (sshService: IoK8sApiCoreV1Service) =>
   sshService?.spec?.ports?.find((port) => parseInt(port.targetPort, 10) === 22)?.nodePort;
 
-export const isTemplateParameter = (value: string): boolean => !!/^\${[A-z0-9_]+}$/.test(value);
+export const isTemplateParameter = (value: string): boolean =>
+  Boolean(/^\${[A-z0-9_]+}$/.test(value));
```

Comment:
Does `Boolean` do something different than `!!`? `!!` converts any other type to a boolean type. I assume that's all that's being done by `Boolean` here, right?

Reply:
@pcbailey Right. Unfortunately, something has changed the way that the whole files are tested here on github, not only diffs, and from now it does not like `!!`. Not sure why... And it always suggests me to use `Boolean(...)` instead of `!!`. Without such changes, the PR would be unmergeable.

Reply:
@hstastna That's strange since it's a valid JavaScript operator. Maybe @yaacov will have an idea of why it's blocking it.